content-src/components/DiscoveryStreamComponents/Navigation/Navigation.jsx

```diff
@@ -25,7 +26,11 @@ export class Navigation extends React.PureComponent {
     const header = this.props.header || {};
     return (
       <div className={`ds-navigation ds-navigation-${alignment}`}>
-        {header.title ? <div className="ds-header">{header.title}</div> : null}
+        {header.title ? (
+          <div className="ds-header">
+            <FluentOrText message={header.title} />
```

Comment:
This results in a div with a span inside. You can have `FluentOrText` set the data-l10n-id on the div instead by putting it as the child instead.

Reply:
Thanks, not sure if I follow, is there an example I can see?

Reply:
https://github.com/mozilla/activity-stream/blob/f5ff7c676dccbf94b660f397b5b0b09e8d43a992/content-src/components/Sections/Sections.jsx#L303-L305

Reply:
That's pretty wild. Updated.